common-voice/sieputra/sieputra.md

```diff
@@ -0,0 +1,11 @@
+<!-- copy format isi ini di foldermu -->
+
+## Hacktoberfest 2022
+
+
+| | Nama/Username common voice | Tanggal | Jam | Jumlah klip | |
+| - | :-: | :-: | :-: | :-: | :-: |
+| | sieputra | 22/10/2022 | 22.00 | ~80 Klip |
```

Comment:
Jumlah klip terlihat di chat bar masih di bawah 60 aktivitas

Reply:
updated in next commit , txs :-)